boards/riscv/it8xxx2_evb/doc/index.rst

```diff
@@ -60,34 +59,159 @@ currently supports the following hardware features:
    * - GPIO
      - on-chip
      - gpio
+   * - ADC
+     - on-chip
+     - adc
+   * - I2C
+     - on-chip
+     - i2c
+   * - KSCAN
+     - on-chip
+     - kscan
+
+
+Other hardware features are not currently supported by Zephyr (at the moment)
+
+The default configuration can be found in the
+:zephyr_file:`boards/riscv/it8xxx2_evb/it8xxx2_evb_defconfig` Kconfig file.
+
+Programming and debugging on it83202
+************************************
+
+Wiring
+=======
+#. Connect your DownLoad Board to your host computer using the USB cable.
```

Comment:
Please use a consistent name  for Download Board - you have the following variations:
"DownLoad Board"
"Download Board"
"download board"

Reply:
Fixed.